docs/MSPX.md

```diff
@@ -3,14 +3,17 @@
 ([back to main page](../README.md))
 
 mLRS provides the MspX technology, which is designed to improve the over-the-air communication for systems using the MSP protocol, specifically INAV. It includes the following features:
-- MSP to CRSF message conversion which provides telemetry elements to the radio and therefore enables Lua scripts on the radio such as the INAV app or telemetry widgets to function.
+- MSP to CRSF message conversion which provides telemetry elements to the radio and therefore enables Lua scripts on the radio such as the INAV Telemetry Widget, FM2M Toolbox or the Horus Mapping Widget to function.
 - Robust framing and parsing which reduces packet losses to a minimum, and compression of some very large MSP messages to increase probability of successful transmission.
+- Provides comprehensive link statistics to the flight controller, enabling them to be shown on the OSD and recorded in the blackbox.
+- Possible to have both RC and MSP serial data on a single UART. 
 
 ***Notes***: 
-- MspX is not strictly compliant with the MSP protocol, and softwares which require strict compliance may not work properly. This includes e.g. MWP. In these cases, mLRS can be set into transparent mode by setting "Rx Ser Link Mode" = "transp." (the below features of MspX will then not be avaiable).
-- MspX is tested and verified for INAV 7.1. It has been successfully tested with INAV 8.0 development builds (it may also work with INAV 6.0 and newer, but this has not been tested).
+- For INAV Versions before 8, MspX can not fully comply with MSP functional design rules and might not work properly with GCS Applications. This includes e.g. MWPTools. In these cases, mLRS can be set into transparent mode by setting "Rx Ser Link Mode" = "transp." (the below features of MspX will then not be avaiable).
```

Comment:
Suggested tweaks:

- 'For INAV 8 and earlier, MspX does not fully...'
- '...GCS applications.' (lowercase applications)